### PR TITLE
connectivity check for ipsec

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/10326

### CODE REVIEW

File: infra-templates/ipsec/15/rancher-compose.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 .catalog:
   name: "Rancher IPsec"
-  version: "0.2.1"
-  minimum_rancher_version: v1.6.11-rc1
```
Comment (deniseschannon) [inline infra-templates/ipsec/15/rancher-compose.yml]: don't remove the min version, update to `v1.6.13-rc1`

File: infra-templates/ipsec/15/docker-compose.yml.tpl [hunk 2]
```diff
@@ -41,6 +41,11 @@ services:
       net.ipv4.conf.default.send_redirects: '0'
       net.ipv4.conf.eth0.send_redirects: '0'
       net.ipv4.xfrm4_gc_thresh: '2147483647'
+  hc:
+    image: leodotcloud/self-health-status:dev
```
Comment (deniseschannon) [inline infra-templates/ipsec/15/docker-compose.yml.tpl]: what is this image going to be?

File: infra-templates/ipsec/15/docker-compose.yml.tpl [hunk 3]
```diff
@@ -12,7 +12,7 @@ services:
       - 500:500/udp
       - 4500:4500/udp
     labels:
-      io.rancher.sidekicks: router
+      io.rancher.sidekicks: router,hc
```
Comment (deniseschannon) [inline infra-templates/ipsec/15/docker-compose.yml.tpl]: let's not abbreviate the service. let's make it a name that people understand

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: infra-templates/ipsec/15/docker-compose.yml.tpl (hunk 3 above)
Comment (leodotcloud) [reply infra-templates/ipsec/15/docker-compose.yml.tpl]: sure, will change it to `healthcheck`

File: infra-templates/ipsec/15/docker-compose.yml.tpl (hunk 2 above)
Comment (leodotcloud) [reply infra-templates/ipsec/15/docker-compose.yml.tpl]: I will package the binary in `rancher/net`

File: infra-templates/ipsec/15/rancher-compose.yml (hunk 1 above)
Comment (leodotcloud) [reply infra-templates/ipsec/15/rancher-compose.yml]: ok

**Comment by deniseschannon** [comment]
LGTM

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**